css/main.css

```diff
@@ -325,3 +325,102 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact-section hr {
+  width: 100% !important;
+}
+
+.contact-section h2 {
+  font-size: 30px;
+}
+
+.contact-section input {
+  border: none;
+  border-bottom: 1px solid #666;
+  outline: none;
+  padding: 10px 0;
+  margin-bottom: 15px;
+}
+
+.contact-section textarea {
+  background: #fbf8f7;
+  width: 100%;
+  padding: 10px 0;
+  margin-bottom: 15px;
+}
+
+.main-contact {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  gap: 100px;
+  padding: 10%;
+}
+
+.contact-left {
+  flex: 1;
+}
+
+.contact-right {
+  flex: 2;
+}
+
+form input[type=text] {
+  width: 49%;
+}
+
+form input[type=email] {
+  width: 100%;
+}
+
+/* .ml {
+  margin-left: 20px;
+} */
```

Comment:
- [ ] [Optional] It would be nice to keep our codebase clean by removing commented codes